src/components/CalendarMonthGrid.jsx

```diff
@@ -60,6 +64,19 @@ const defaultProps = {
   monthFormat: 'MMMM YYYY', // english locale
 };
 
+function getCalendarMonthGridWidthStyle(orientation, withWeekNumbers) {
```

Comment:
Yeah that totally makes sense! :) Can we pull this out as well into a util method? I'm very much about small utility modules wherever possible. It makes testing a dream.